### ISSUE TITLE
Poor documentation

### ISSUE THREAD

**Issue by zecar** [mceSystems/libijs#1]
I want to try this package but I keep getting "Couldn't find device"

**Comment by MCE-KobyBo** [comment]
Yeah we're still missing some services\tools and proper documentation, but we're working on it.

As for your issue, could you give me some more details about your device and your environment:
- Which OS are you using?
- If you're not using Windows, what usbmuxd are you using?
- Did you get this error while trying one of the tools or did you write your own code?
- What device did you test it with?
